torch/csrc/jit/serialization/export.cpp

```diff
@@ -113,13 +113,13 @@ void validateBlock(
           WithInsertPoint guard(node);
           auto* new_node =
               b->owningGraph()->insertNode(b->owningGraph()->create(
-                  Symbol(::c10::onnx::ATen),
+                  Symbol(::c10::aten::ATen),
                   node->inputs(),
                   node->outputs().size()));
           for (size_t i = 0; i < node->outputs().size(); ++i) {
             node->output(i)->replaceAllUsesWith(new_node->output(i));
           }
-          new_node->s_(Symbol::fromQualString("attr::operator"), "expand");
+          new_node->s_(Symbol::fromQualString("attr::operator_name"), "expand");
```

Comment:
nit: attr::operator_name

Reply:
What is the issue? I don't see the typo

Reply:
using `attr::operator_name` directly?

Reply:
There are 2 ways to get those attributes.

The indirect through `fromQualString("attr::operator")` and the direct using `::c10::attr::operator`. The former works for `operator` while the latter doesn't. All other attributes support both, so I renamed this for consistency in a BC way